tests/integration/util_insert.go

```diff
@@ -120,97 +114,27 @@ func NewVarCharSameFieldData(fieldName string, numRows int, value string) *schem
 }
 
 func NewStringFieldData(fieldName string, numRows int) *schemapb.FieldData {
```

Comment:
Just remove this function?

Reply:
direct removal will cause too many code change